airflow/dags/news_crawling_to_db_dag.py

```
@@ -0,0 +1,51 @@
+# 1) 링크 수집
+def task_get_article_links(**context):
+    from libs.news_article.crawler.linkCrawling import get_article_link
+    links = get_article_link()
+    return links
+
+# 2) Articles 테이블에 적재/업서트
+def task_upsert_articles(**context):
+    from libs.news_article.db_update.article_db_handler import get_db_connection, process_crawled_articles
+    ti = context["ti"]
+    links = ti.xcom_pull(key="return_value", task_ids="get_article_links")
+    conn = get_db_connection()
```

Ruff (0.12.2)

Local variable `conn` is assigned to but never used

Remove assignment to unused variable `conn`

(F841)